tests/seg/test_evaluation.py

```diff
@@ -0,0 +1,60 @@
+import itk
+import numpy as np
+from typing import Sequence
+from segmantic.prepro.core import Image3
+
+import pytest
+from segmantic.seg import evaluation
+
+
+def make_image(
+    shape: Sequence[int], spacing: Sequence[float], value: int = 0
+) -> Image3:
+    """Create image with specified shape and spacing"""
+    assert len(shape) == len(spacing)
+    dim = len(shape)
+
+    region = itk.ImageRegion[dim]()
+    region.SetSize(shape)
+    region.SetIndex(tuple([0] * dim))
+
+    image = itk.Image[itk.UC, dim].New()
+    image.SetRegions(region)
+    image.SetSpacing(spacing)
+    image.Allocate()
+
+    image[:] = value
+    return image
+
+
```

Comment:
if you put ``labelfield`` as a fixture, you can reuse it in different tests

```python
def labelfield() -> nd.array:
     return make_image(shape=(10, 10), spacing=(1.0, 1.0), value=0)

def test_confusion_matrix(labelfield: nd.array):
    ...
    
def test_hausdorff(labelfield: nd.array):
   ...    
    
```

Reply:
probably. I will move it to a fixture module in `tests`

Reply:
done